src/components/gw-user-options.vue

```diff
@@ -20,30 +20,19 @@
         </template>
         <template slot="body">
             <span class="dropdown-title">My Profile</span>
-            <router-link :to="{ name: 'settingsUsersProfile' }" class="dropdown-item">
-                <span>Users Settings</span>
-            </router-link>
-            <router-link
-                v-if="$can('company-settings', 'settingsmenu')"
-                :to="{ name: 'settingsCompaniesProfile' }"
-                class="dropdown-item"
+            <template
+                v-for="option in userOptionsList" 
```

Comment:
This is not needed. All you need to do is verify if an option is in the mapper to use that instead of the default.

Reply:
That's what I am doing in ln line 99. But I am doing one time in code rather than 4 times